### ISSUE TITLE
Ensure the focus order in Clarity wizard is logical

### ISSUE THREAD

**Issue by bashibozuk** [vmware/clarity#4674]
## Describe the bug

The close button of the Clarity wizard is placed after the left hand navigation in the focus order. Visually, and in consistence with other dialogs, this close button should be first in the dialog's focus order.
## How to reproduce

https://clarity.design/documentation/wizards

1. Put focus outside the wizard
2.  Press Tab
3.  The focus is in the wizard navigation
4.  Press Tab
5. The focus goes to the close button
6. Press tab
7. The focus goes to the wizard main area

## Expected behavior

The focus order should follow the visual order of elements

## Versions

**App**

- Angular: [9.0.2]
- Node: [12]
- Clarity: [v3]

**Device:**

- Type: [MacBook]
- OS: [macOS]
- Browser [chrome]
- Version [81.0.4044.138]

## Additional notes

_Add any other notes about the problem here._

**Comment by chlane** [comment]
This is covered in https://github.com/vmware/clarity/issues/4834

**Comment by bdryanovski** [comment]
I think that this will create a breaking change, or we have to keep two working copies of both HTML and styles - we had the same problem with the Modals windows and their order. 

I think that this will be better resolve in the core implementation.

**Comment by Jinnie** [comment]
I see Chris has kindly added this issue to the internal A11Y tracking.
Closing it here, to avoid duplication.
